python/ray/tune/utils/placement_groups.py

```diff
@@ -120,9 +122,14 @@ def __init__(self,
         self._bind()
 
     @property
-    def head_cpus(self):
+    def head_cpus(self) -> Optional[float]:
         return self._bundles[0].get("CPU", None)
 
+    @property
+    def bundles(self) -> List[Dict[str, float]]:
```

Comment:
Is there anyway to annotate these APIs (`bundles` and `head_cpus`) to be internal APIs?